docs/grpc.md

```
@@ -326,12 +326,20 @@ Returns all tenants.
 
 | Request | Response |
 | ------- | -------- |
 | [`GetTenantRequest`](#gettenantrequest) | [`Tenant`](#tenant) |
 
+#### RemoveTenant
+
+Removes a tenant. Only allowed if the tenant has no associated wallets.
+
+| Request | Response |
+| ------- | -------- |
+| [`RemoveTenantRequest`](#removetenantrequest) | [`.google.protobuf.Empty`](#.google.protobuf.empty) |
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)